dll/win32/shell32/dialogs/folder_options.cpp

```diff
@@ -192,10 +192,13 @@ HBITMAP CreateRadioMask(HDC hDC)
 
 // CMSGlobalFolderOptionsStub --- The owner window of Folder Options.
 // This window hides taskbar button of Folder Options.
+
+#define GlobalFolderOptionsClassName _T("MSGlobalFolderOptionsStub")
```

Comment:
If we unconditionally compile the shell32 in unicode, you could directly use the `L"...."` syntax.

Reply:
`_T` was already there, I just moved it so I could use it with `FindWindow.